Sources/Extensions/UIKit/UIWindowExtensions.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  UIWindowExtensions.swift
+//  SwifterSwift
+//
+//  Created by Omar Albeik on 6/2/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if canImport(UIKit)
+import UIKit
+
+#if os(iOS)
+public extension UIWindow {
+
+    /// SwifterSwift: Switch current root view controller with a new view controller.
+    ///
+    /// - Parameters:
+    ///   - viewController: new view controller.
+    ///   - animated: set to true to animate view controller change (default is true).
+    ///   - duration: animation duration in seconds (default is 0.5).
+    ///   - options: animataion options (default is .transitionFlipFromRight).
+    ///   - completion: optional completion handler called after view controller is changed.
+    public func switchRootViewController(
+        to viewController: UIViewController,
+        animated: Bool = true,
+        duration: TimeInterval = 0.5,
+        options: UIViewAnimationOptions = .transitionFlipFromRight,
+        _ completion: (() -> Void)? = nil) {
```

Comment:
Nit: You don't need to omit the external argument here because most people would use this as a trailing closure anyways